### PR TITLE
Update db schema after rails upgrade

### PR DESCRIPTION
**Pull request by JonathanHallam**
An update to rails means that there's changes to the db schema that
can only be seen when a migration is run. This means that these changes
will be in the schema rather than being drawn in with an upcoming
migration

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -46,10 +46,11 @@
     t.string "action_link_url"
     t.string "action_link_content"
     t.string "action_link_summary"
+    t.string "sub_heading"
```
Comment (leenagupte) [inline db/schema.rb:49]: I don't think this should be here. I think you need to rollback the last migration first.

Comment (kevindew) [inline db/schema.rb:49]: Looks like you've got this from your new migration. You'll have to roll that back first (or do db:reset on the main branch if you're not precious about the data)

**Review by leenagupte — COMMENTED** [review]
One small issue.

File: db/schema.rb (hunk 1 above)
Comment (kevindew) [reply db/schema.rb:49]: Jinx 😀

File: db/schema.rb [hunk 2]
```diff
@@ -49,18 +49,17 @@
...
 
-  create_table "coronavirus_timeline_entries", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+  create_table "coronavirus_timeline_entries", charset: "utf8", force: :cascade do |t|
     t.bigint "coronavirus_page_id", null: false
     t.text "content", null: false
     t.string "heading", null: false
     t.integer "position", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.string "national_applicability"
```
Comment (kevindew) [inline db/schema.rb:59]: I don't think you are meaning to lose this - you may have gone back too far?

Comment (JonathanHallam) [reply db/schema.rb:59]: 😆 how is this happening!

Comment (leenagupte) [reply db/schema.rb:59]: Beat me to it!

**Review by leenagupte — APPROVED** [review]

File: db/schema.rb (hunk 2 above)
Comment (JonathanHallam) [reply db/schema.rb]: Okay, now looking healthy 😬

**Pull request merged**